docs/user-guide.md

```diff
@@ -15,37 +15,35 @@ Such annotations are useful to (for example):
 
 ## Java variables are references
 
-In Java, all variables, except primitives like `int`, are references. We often
-think of `String x;` as meaning "`x` is a `String`", but actually it means "`x` is a
-_reference_ to a `String`". One important difference between these two views is that
-`x` can also be null. The goal of JSpecify is to move towards a world where `String
-x;` means "`x` is a reference to an actual string", and if you want a reference
-that might be null then you write `@Nullable String x;`.
+In Java, all non-primitive variables are references. We often think of `String
+x;` as meaning "`x` is a `String`", but actually it means "`x` is a _reference_
+to a string or it is null". JSpecify includes a notion of "null-marked code"
+where `String x;` means "`x` is a reference to an actual string", and if you
+want a reference that might be null then you write `@Nullable String
+x;`. Null-marked code is simply code that is covered by the `@NullMarked`
+annotation.
 
 ## Types and nullness
 
-In the JSpecify world, most references have one of three possible properties
+In the JSpecify world, references have one of three possible properties
 regarding nullness:
 1. They can be null.
 2. They can't be null.
-3. We don't know if they can be null or not.
+3. JSpecify annotations haven't indicated whether they can be null.
```

Comment:
Is it simpler to remove the reference to "the JSpecify world" and say something like:

Each reference can have one of three possible properties regarding nullness:
1. JSpecify annotations indicate that it can be null.
2. JSpecify annotations indicate that it can't be null.
3. JSpecify annotations don't indicate whether it can be null.

Reply:
Yes, I think that's better.